client/components/Events/EventDateTime.tsx

```diff
@@ -50,9 +62,15 @@ export class EventDateTime extends React.PureComponent<IProps> {
             );
         }
 
+        const noEndTime = item.dates?.no_end_time;
+        const allDay = item.dates?.all_day;
```

Comment:
Yes, I thought the same but isAllDay flag is not same as allDay. isAllDay is build according to previous needs when we didn't have no_end_time falg.

Reply:
yep I know, but we should harmonize that probably because both should have the same effect, if not we should change the names because it's just confusing